draft-ietf-masque-connect-ip.md

```diff
@@ -174,24 +210,29 @@ optional, and have default values if not included.
 The defined variables are:
 
 target:
-: The variable "target" contains a hostname or IP address of a specific host to
-which the client wants to proxy packets. If the "target" variable is not
-specified, the client is requesting to communicate with any allowable host. If
+: The variable "target" contains a DNS hostname (reg-name), IPv6 literal
+(IPv6address) or IPv4 literal (IPv4address) address ({{URI}} syntax elements
+within parenthis) of a specific host to which the client wants to proxy
+packets. If the "target" variable is not specified or its value is "\*", the
+client is requesting to communicate with any allowable host. If
 the target is an IP address, the request will only support a single IP version.
 If the target is a hostname, the server is expected to perform DNS resolution
 to determine which route(s) to advertise to the client. The server SHOULD send
 a ROUTE_ADVERTISEMENT capsule that includes routes for all addresses that were
 resolved for the requested hostname, that are accessible to the server, and
 belong to an address family for which the server also sends an ADDRESS_ASSIGN
-capsule.
+capsule. Note that IPv6 scoped addressing zone identifiers are not supported.
 
 ipproto:
-: The variable "ipproto" contains an IP protocol number, as defined in the
-"Assigned Internet Protocol Numbers" IANA registry. If present, it specifies
+: The variable "ipproto" contains an IP protocol number, as defined in
+the "Assigned Internet Protocol Numbers" IANA registry. If present, it specifies
 that a client only wants to proxy a specific IP protocol for this request. If
-the value is 0, or the variable is not included, the client is requesting to
-use any IP protocol.
-{: spacing="compact"}
+the value is "\*", or the variable is not included, the client is requesting to
+use any IP protocol.  {: spacing="compact"}
+
+Also note that this URI Template expansion requires using
```

Comment:
Let's remove this note for now since it's incorrect. If we find out that there are implementations issues with `:` or `*` then we can add a note specifically about those